media/js/src/editors/OptimalChoiceConsumption.jsx

```diff
@@ -0,0 +1,220 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { MathJaxProvider, MathJaxFormula } from 'mathjax3-react';
```

Comment:
The graph type looks good, but this will need to be changed to:
```
import { MathJax } from 'better-react-mathjax';
```

I would rebase this branch on top of the current master branch in order to get the latest version of our package.json.. or a merge from master. But also I am happy to do that work too.

Reply:
Oh that works so much better now. It was super slow before.